docs/ocean/design_docs/vert-lagrang-remap/vert-lagrang-remap.rst

```diff
@@ -0,0 +1,124 @@
+
+Vertical Lagrangian-remap method
+======================
+
+date: 2020/11/24
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+
+
+Summary
+-------
+
+The vertical Lagrangian-remap method is to be implemented to support the choice 
+of hybrid vertical coordinates. It can be seen as an extension of the Arbitrary 
+Lagrangian-Eulerian method which is already implemented in MPAS-Ocean, a key 
+difference being that the solution of diasurface transport is coordinate-free. 
+This has the advantage of removing the vertical CFL constraint, which is of 
+particular concern for an evolving ice shelf geometry in terrain-following 
+coordinate schemes. Generalization of the vertical coordinate in the horizontal 
+equations of motion is not in the scope of this development. This development 
+will include some infrastructure for specifying the target grid for regridding. 
+The minimum criteria for success are not degrading the accuracy of the solution 
+or the performance by more than [an acceptable threshold] for several basic test 
+cases... .
```

Comment:
I think we can leave the threshold ambiguous for now.  We can also fill in which test cases we will use for checking accuracy as the design progresses.  Probably ISOMIP+ Ocean0 would be one, maybe the baroclinic channel?  We can ask @mark-petersen which tests he would suggest.